packages/taquito-utils/test/format.spec.ts

```diff
@@ -21,4 +22,23 @@ describe('Format', () => {
   it('Should convert tz to mutez', () => {
     expect(format('tz', 'mutez', 1)).toEqual(new BigNumber(1000000));
   });
+
+  it('Should be valid bytes tezostaquito example', () => {
+    const formattedInput: string = [
+      'Tezos Signed Message:',
+      'some url',
+      '18:43:34Z',
+      'something',
+    ].join(' ');
+
+    const bytes = char2Bytes(formattedInput);
+    const bytesLength = (bytes.length / 2).toString(16);
+    const addPadding = `00000000${bytesLength}`;
+    const paddedBytesLength = addPadding.slice(addPadding.length - 8);
+    const payloadBytes = '05' + '01' + paddedBytesLength + bytes;
+    const regex = new RegExp('^(0x|0X)?[a-fA-F0-9]+$');
+    const check = regex.test(payloadBytes);
+    expect(check).toEqual(true);
```

Comment:
I'm not sure this actually tests the functionality properly. The test seems to just go through the steps in the util method. A better assertion would be to re-convert the bytes to string successfully with the same value retained.